common/app/conf/switches/CommercialSwitches.scala

```diff
@@ -9,24 +9,27 @@ trait CommercialSwitches {
     SwitchGroup.Commercial,
     "dfp-caching",
     "Have Admin will poll DFP to precache adserving data.",
+    owners = Seq(Owner.withName("commercial team")),
     safeState = On,
     sellByDate = never,
     exposeClientSide = false
   )
 
   val HeaderBiddingUS = Switch(
-     SwitchGroup.Commercial,
-     "header-bidding-us",
-     "Auction adverts on the client before calling DFP (US edition only)",
-     safeState = Off,
-     sellByDate = never,
-     exposeClientSide = true
+    SwitchGroup.Commercial,
+    "header-bidding-us",
+    "Auction adverts on the client before calling DFP (US edition only)",
+    owners = Seq(Owner.withGithub("regiskuckaertz ")),
+    safeState = Off,
+    sellByDate = never,
```

Comment:
Switches that never expire mean something (and are probably overused). The default behaviour should probably be to delete the switch (but leave the code in place) apart from a few switches that are used in panic situations.

If you plan on switching something off it should have a sensible `sellByDate`.

Maybe create a `NeverExpiresSwitch(...)` instead of `sellByDate = never` and at most these should belong to a team and not an individual.